### PR TITLE
[model compression] reorganize quantizer files

### PR DESCRIPTION
**Pull request by chenbohua3**

### CODE REVIEW

**Comment by liuzhe-lz** [comment]
Suggest to put algorithms in [nni.algorithms.compression](https://github.com/microsoft/nni/tree/master/nni/algorithms/compression)

**Comment by chenbohua3** [comment]
> Suggest to put algorithms in [nni.algorithms.compression](https://github.com/microsoft/nni/tree/master/nni/algorithms/compression)

I am a little confused about this. The quantization algorithms are already in nni/algorithms/compression/pytorch/quantization/*

File: nni/algorithms/compression/pytorch/quantization/__init__.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-from .quantizers import *
+from .bnn_quantizer import BNNQuantizer
+from .dorefa_quantizer import DoReFaQuantizer
+from .lsq_quantizer import LsqQuantizer
+from .nativa_quantizer import NaiveQuantizer
```
Comment (J-shang) [inline nni/algorithms/compression/pytorch/quantization/__init__.py:7]: nativa?

**Comment by J-shang** [comment]
I didn't look at the details of each split file, is this just a split of the large file?

File: nni/algorithms/compression/pytorch/quantization/__init__.py (hunk 1 above)
Comment (chenbohua3) [reply nni/algorithms/compression/pytorch/quantization/__init__.py]: Have corrected it to native_quantizer

**Comment by chenbohua3** [comment]
> I didn't look at the details of each split file, is this just a split of the large file?

Yes

**Review by J-shang — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**